packages/atomic/src/components.d.ts

```diff
@@ -882,6 +882,36 @@ export namespace Components {
          */
         "mobileBreakpoint": string;
     }
+    interface AtomicSegmentedFacet {
+        /**
+          * Specifies a unique identifier for the facet.
+         */
+        "facetId"?: string;
+        /**
+          * The field whose values you want to display in the facet.
+         */
+        "field": string;
+        /**
+          * Whether to exclude the parents of folded results when estimating the result count for each facet value.
+         */
+        "filterFacetCount": boolean;
+        /**
+          * The maximum number of results to scan in the index to ensure that the facet lists all potential facet values. Note: A high injectionDepth may negatively impact the facet request performance. Minimum: `0`
+         */
+        "injectionDepth": number;
```

Comment:
Just for readability:

```suggestion
          * The maximum number of results to scan in the index to ensure that the facet lists all potential facet values.
          * Note: A high injectionDepth may negatively impact the facet request performance.
          * Minimum: `0`
         */
        "injectionDepth": number;
```

Also, does it have any `* @defaultValue` ?

If it does, let's add it 😇